packages/core/security/core-security-server/src/contracts.ts

```diff
@@ -6,6 +6,7 @@
  * Side Public License, v 1.
  */
 
+import { CoreFipsService } from './fips';
```

Comment:
NIT: `import type`

Reply:
Fixed in https://github.com/elastic/kibana/pull/181187/commits/7b7ed010889b5072d430622be372d4af9bfee487